### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gerald-byrket**

### CODE REVIEW

**Comment by jpwhite4** [comment]
Suggest adding a `Requires: ondemand >= 2.0.0` to the RPM spec file this will prevent admins installing it by mistake on an ondemand 1.8 system.

**Comment by jpwhite4** [comment]
The path to the Centos 7 / RedHat 7 RPM looks like it could be incorrect. I would expect the RPM filename to have `el7` in the suffix and not `el8`.

Also will the RPM be hosted in the OSC yum repo? If an admin has installed ondemand following the instructions then they would already have the OSC yum repo available and could just do a `def install ondemand-bc_access_menu`

File: packaging/ondemand-bc_access_menu.spec [hunk 1]
```diff
@@ -21,7 +21,7 @@ License:  MIT
 URL:      https://github.com/OSC/%{repo_name}
 Source0:  https://github.com/OSC/%{repo_name}/archive/%{git_tag}.tar.gz
 
-Requires: ondemand
+Requires: ondemand => 2.0.0
```
Comment (jpwhite4) [inline packaging/ondemand-bc_access_menu.spec]:
```suggestion
Requires: ondemand >= 2.0.0
```

**Comment by gerald-byrket** [comment]
> Also will the RPM be hosted in the OSC yum repo? If an admin has installed ondemand following the instructions then they would already have the OSC yum repo available and could just do a `def install ondemand-bc_access_menu`

FYI. I tried the following.

1)	Uninstall the menu using dnf remove
2)	Ran dnf installed command, but the menu couldn’t be found.
3)	Ran dnf provides, but couldn’t be found.

The only way I could get it to install was by using the rpm.

**Pull request merged**